src/Effects.js

```diff
@@ -627,61 +715,54 @@ module.exports = class Effects {
       }
     };
 
-    this.falling_poop = {
-      poop: [],
-      size: 50,
+    this.smiling_poop = {
+      poopList: [],
       init: function () {
-        for (let i = 0; i < 20; i++) {
-          this.poop.push({
+        for (let i = 0; i < 6; i++) {
+          this.poopList.push({
             x: randomNumber(10, 390),
-            y: -50,
+            y: randomNumber(10, 390),
             rot: randomNumber(0, 359),
-            speed: p5.random(1,5),
+            life: 5,
           });
         }
       },
-      update: function () {
-        this.size += randomNumber(-5, 5);
-      },
       draw: function () {
-        if (this.poop.length < 1) {
+        if (this.poopList.length < 1) {
           this.init();
         }
-        for (let i = 0; i < this.poop.length; i++) {
+        for (let i = 0; i < this.poopList.length; i++) {
           p5.push();
-          const poop = this.poop[i];
+          const poop = this.poopList[i];
           p5.translate(poop.x, poop.y);
           p5.rotate(poop.rot);
-          p5.scale(2, 2);
-          drawFallingPoop(p5._renderer.drawingContext);
-          p5.pop();
-          poop.y += poop.speed;
-          poop.rot++;
-          if (poop.y > 410) {
+          p5.scale(poop.life / 20);
+          p5.drawingContext.globalAlpha - 0.5;
+          drawPoop(p5._renderer.drawingContext);
```

Comment:
Another place we haven't switched to draw-once.  I assume this is because we reported about ~23fps on this effect before and we've cut the number of emoji being drawn by 70%.  This might be 'good enough' but remember that those perf tests are from _just_ running the background, with no dancers onscreen.  If we can drive the cost of the background down further we should.